### DISCUSSION TITLE
Making Siri Announce Notifications. Making Notifications "Time Sensitive"

### DISCUSSION THREAD

**Discussion by csharpcoder198** [shinyorg/shiny#1025] [Ideas]
I am wondering how to implement "Have Siri Announce Notifications" I have checked earlier it sounded like it is not already implemented.

From an iOS perspective I will need to Configure "myShinyApp"
according to iOS instructions here:
[Instructions from Apple](https://support.apple.com/guide/iphone/have-siri-announce-calls-and-notifications-iph838fd6fd4/ios)

Is there some part of Shiny I need to Override a Shiny Notification Type in order to make it "Time Sensitive" as the instructions allude.

![image](https://user-images.githubusercontent.com/105575743/171519089-ac8db489-a8fa-443e-a6ba-7858f7d5323f.png)

**Comment by aritchie** [comment]
The siri part is not anything that can be done with the platform API so shiny doesn't do anything there.

Time sensitive notifications are automatically requested by Shiny.Notifications if you set a schedule time and call: 

https://github.com/shinyorg/samples/blob/main/Notifications/Sample/Create/CreateViewModel.cs#L66

**Reply by csharpcoder198** [reply]
Awesome where do I schedule a time?

**Reply by csharpcoder198** [reply]
Oh no I see you mean not schedule a Time with you but Schedule Time for the notification.

**Reply by aritchie** [reply]
[notification.Schedule](https://github.com/shinyorg/samples/blob/main/Notifications/Sample/Create/ScheduleViewModel.cs#L28)

The sample has pretty much every feature for what local notifications uses
